### PR TITLE
Leaderboard page now auto-reloads if there are changes to the database

### PR DESCRIPTION
**Pull request by GV14982**
Built an API route for getting the leaderboard.
Wrote a short script that saves the current state on page load, then compares that to the data from the API route every 5 seconds.

### CODE REVIEW

**Comment by GV14982** [comment]
I also want to mention that I am currently using a kind of hacky approach to storing the current state of the leaderboard to be able to access it from the view's js.

I am doing something [similar to this](https://stackoverflow.com/questions/10919650/accessing-express-js-local-variables-in-client-side-javascript), but I think we need to figure out a better way to handle it:

**Comment by GV14982** [comment]
Also #10

File: public/js/getUpdate.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { href, protocol, hostname, port, pathname } = window.location;
```
Comment (seanmetzgar) [inline public/js/getUpdate.js:1]: This is f—ing beautiful!

File: public/js/getUpdate.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const { href, protocol, hostname, port, pathname } = window.location;
+const newBoard = document.getElementById('hidden');
+url = `${protocol}//${hostname}:${port}/api/v1`;
+axios.defaults.baseURL = url;
+
```
Comment (seanmetzgar) [inline public/js/getUpdate.js:5]: is axios a node xhr library?

**Review by seanmetzgar — COMMENTED** [review]
I’ll check this out tomorrow on my laptop and give it a thorough test

**Pull request merged**